web-local/tests/olap-connectors.spec.ts

```diff
@@ -1,15 +1,21 @@
 import { expect } from "@playwright/test";
 import { test } from "./setup/base";
-
-/**
- * These tests focus on form validation and YAML generation for OLAP connectors.
- * They don't test connections to live OLAP databases - that's handled by runtime integration tests.
- * Future work: Add tests using ClickHouse testcontainers for end-to-end validation.
- */
+import { ClickHouseTestContainer } from "./utils/clickhouse";
 
 test.describe("ClickHouse connector", () => {
   test.use({ project: "Blank" });
 
+  const clickhouse = new ClickHouseTestContainer();
+
+  test.beforeAll(async () => {
```

Comment:
Would be good to move this to a static method on ClickHouseTestContainer that also returns the instance.

Reply:
I can't quite tell exactly what you have in mind. But regarding another layer of abstraction– that might be valuable, but I'd prefer not to add extra abstractions here before we know how our usage of this class will evolve. For example, I could especially see the `seed` method being parameterized based on which dataset you want to use for a given test suite.

Reply:
We could do something like `ClickHouseTestContainer.use(<seed param>)`. But lets do that in a follow up when we use more of ClickHouseTestContainer